src/lib.rs

```diff
@@ -160,11 +160,13 @@ impl Mediawiki {
         if !image["missing"].is_null() {
             return Ok(None);
         }
-        let url = image["imageinfo"][0]["url"]
+        let lossy_url = image["imageinfo"][0]["url"]
             .as_str()
             .ok_or_else(|| Error::Json(image.clone()))?;
+        let url = format!("{}{}format=original", lossy_url,
```

Comment:
It looks like we can probably add a line like this to make this simpler:

```rust
        request.arg("format", "original");
```

Probably add it around https://github.com/FTB-Gamepedia/mediawiki-rs/pull/8/changes#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R152-R153 if I were to choose, but anywhere before the actual request happens should be fine.